install.ps1

```diff
@@ -38,6 +38,7 @@ $ErrorActionPreference = "Stop"
 $repo = "onflow/flow-cli"
 $versionURL = "https://api.github.com/repos/$repo/releases/latest"
 $assetsURL = "https://github.com/$repo/releases/download"
+$stableVersion = "v1.20.5" # The specific stable version to download
```

Comment:
Might be better to query the GitHub API and fetch the latest non pre-release (`prerelease == false`) using `Get-Version`

Reply:
I tried making the change, but unfortunately, grep prerelease false doesn't return the correct entry. Also, we are trying to code freeze this last version now, so I don't think hardcoding is too bad. I agree the other way is more ideal, but to be honest, I'm a little out of my depth here with getting this and the corresponding PowerShell script to work. However, the current hardcoded version scripts seem to work. Feel free to take a shot if you think you can get it to work without hardcoding.

Reply:
Improved the PowerShell script in [bdf33f5](https://github.com/onflow/flow-cli/pull/1663/commits/bdf33f5bb6148e24b995901203c8aeb256a996bc).

Will improve the Shell script now

Reply:
Updated the shell script in [0fa235d](https://github.com/onflow/flow-cli/pull/1663/commits/0fa235da235ee62241ce58955bfe74130c33f72e)